src/Event.h

```diff
@@ -4,14 +4,16 @@
 #include <unordered_map>
 #include <string>
 
+enum class EventType {load, EVENT_COUNT};
```

Comment:
In my experience, all enums values are  effectively constants and should be cap case.